frontend/test/metabase-smoketest/README.md

```diff
@@ -0,0 +1,15 @@
+# Metabase Smoketests
+
+These files exist to ensure that we can always do the things we expect with Metabase. :) 
+
+## Running
+
+From the root of the repository:
+
+- If you already have built Metabase with `./bin/build`, just run this to run all the tests.
+
+```shell
+yarn run test-cypress-no-build --testFiles frontend/test/metabase-smoketest
```

Comment:
Might be nice to add a named task for this so it's easier to remember. e.g. `yarn test-cypress-smoketest`. We configure those in `package.json`.

Reply:
good idea!